swap_meet/vendor.py

```diff
@@ -1,2 +1,69 @@
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__(self, inventory=None):
+        if inventory:
+            self.inventory = inventory
+        else:
+            self.inventory = []
+
+
+    def add(self, added_item):
+        self.inventory.append(added_item)
+        return added_item
+
+
+    def remove(self, removed_item):
+        if removed_item not in self.inventory:
+            return False 
+        else:
+            self.inventory.remove(removed_item)
+            return removed_item  
```

Comment:
Let's remove the `else` and let the main focus of the method stand on its own:

 ```suggestion
        self.inventory.remove(removed_item)
        return removed_item  
```